documentation/asciidoc/computers/raspberry-pi/power-supplies.adoc

```diff
@@ -90,6 +90,11 @@ All Raspberry Pi models with a PoE-capable Ethernet jack require a HAT to draw p
 | 1.6A (600mA if using a 3A power supply)
 | 800mA
 
+| Pi 500+
+| 5.0A
+| 1.6A
+| 800mA (with LEDs off and SSD inactive)
```

Comment:
It's wrong to mention LEDs / SSD here because this is referring to the supply current and not the consumed current

Reply:
This is the table at https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#typical-power-requirements where the 4th column is titled "Typical bare-board active current consumption".